chromadb/test/property/test_add.py

```diff
@@ -188,13 +189,12 @@ def test_add_large(
         metadatas=cast(Metadatas, record_set["metadatas"]),
         documents=cast(List[str], record_set["documents"]),
     ):
-        coll.add(*batch)
+        results = coll.add(*batch)
+        if results["ids"] is None:
```

Comment:
Can `results["ids"]` ever be `None` ? I'm not sure this check is doing the right thing.

Reply:
removed